README.md

```diff
@@ -3,6 +3,11 @@ extension-tao-booklet
 
 An extension for TAO to create test booklets (publishable in MS-Word and PDF along with Answer Sheets)
 
+## Warning
+Due to the move of core libs to `npm` repositories, some code might not work on legacy browsers.
```

Comment:
it's not linked to `npm` but to the update to ES2015

Reply:
yep my bad, will reword that